fiddler/manifest.json

```diff
@@ -0,0 +1,43 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "ee617671-508e-4bb3-ba25-8815b11a16aa",
+  "app_id": "fiddler",
+  "classifier_tags": [
+    "Supported OS::Linux",
+    "Supported OS::Mac OS",
+    "Supported OS::Windows"
+  ],
+  "display_on_public_website": false,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "",
+    "title": "fiddler",
+    "media": []
```

Comment:
Please add some images to your tile - this can include screenshots of your UI, Datadog dashboards, etc. Media can be added with the following syntax:

```
    {
        "media_type": "image",
        "caption": "caption goes here",
        "image_url": "path-to-image.png"
      },
```